Utils/ContextToYML.yml

```diff
@@ -0,0 +1,202 @@
+commonfields:
+  id: ContextToYML
+  version: 2
+name: ContextToYML
+script: |
+  import yaml
+
+  """
+  This script parse a Context output into yml formatted file with the context path of each field.
+  For example:
+  {
+      "id":12131,
+      "description":"desc",
+      "summary":"do-not-delete",
+      "created":"2019-03-25T16:13:13.188+0200",
+      "issuetype":{
+          "id":10004,
+          "name":"Bug"
+      },
+      "project":{
+          "id":10001,
+          "key":"SOC",
+          "projectTypeKey":"software"
+      },
+      "status":{
+          "id":10003,
+              "StatusCategory":{
+                  "key":"new",
+                  "colorName":"blue-gray",
+                  "name":"To Do"
+          }
+      }
+  }
+  ==>
+  arguments: []
+  name: integration-command
+  outputs:
+  - contextPath: Demisto.Id
+    description: ''
+    type: Number
+  - contextPath: Demisto.Description
+    description: ''
+    type: String
+  - contextPath: Demisto.Summary
+    description: ''
+    type: String
+  - contextPath: Demisto.Created
+    description: ''
+    type: String
+  - contextPath: Demisto.Issuetype.Id
+    description: ''
+    type: Number
+  - contextPath: Demisto.Issuetype.Name
+    description: ''
+    type: String
+  - contextPath: Demisto.Project.Id
+    description: ''
+    type: Number
+  - contextPath: Demisto.Project.Key
+    description: ''
+    type: String
+  - contextPath: Demisto.Project.ProjectTypeKey
+    description: ''
+    type: String
+  - contextPath: Demisto.Status.Id
+    description: ''
+    type: Number
+  - contextPath: Demisto.Status.StatusCategory.Key
+    description: ''
+    type: String
+  - contextPath: Demisto.Status.StatusCategory.Colorname
+    description: ''
+    type: String
+  - contextPath: Demisto.Status.StatusCategory.Name
+    description: ''
+    type: String
+  """
+
+  VERBOSE = demisto.getArg('verbose') == 'true'
+  CAMELIZE = demisto.args().get('camelize', 'false') == 'true'
+
+
+  def flatten_json(nested_json):
+      out = {}
+
+      def flatten(x, name=''):
+          # capitalize first letter in each key
+          try:
+              name = name[0].upper() + name[1:] if CAMELIZE else name
+          except IndexError:
+              name = name.title() if CAMELIZE else name
```

Comment:
only reason you will have IndexError is if the string is empty.
in that case, `name.Title()` will have no effect.